Fixed helm test sample.

WilliamDenniss opened the pull request:
The current sample doesn't work as documented.

New sample based on https://github.com/kubernetes/charts/pull/1156.

WilliamDenniss committed:
Fixed helm test sample.

The current sample doesn't work as documented.

New sample based on https://github.com/kubernetes/charts/pull/1156.

thomastaylor312 reviewed (approved):
This is good to merge once https://github.com/kubernetes/charts/pull/1156 is merged. Thank you for improving the example

michelleN commented on docs/chart_tests.md:
```diff
@@ -34,19 +34,33 @@ mariadb/
   templates/
   templates/mariadb-tests.yaml
```
We'll probably want to change this chart structure also to match the example below.

WilliamDenniss replied:
Fixed. Also removed the `LICENSE` reference as that file isn't the chart's folder.

michelleN commented:
oh awesome. I like this example way better. The original example was never meant to work on the actual stable/wordpress chart. It was just meant to show what a test might actually look like and where it might live in a chart. Seeing an actual example makes lots of sense though. We should add a link to the stable/wordpress chart as well in case the example changes in the kubernetes/charts repo. People will be able to see the updated tests if it ends up changing.

technosophos commented:
Can someone provide an update on this one?

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/helm/2499)
<!-- Reviewable:end -->

michelleN commented:
@technosophos I had requested some changes but they were minor. Happy to approve this one and make the one change I had in mind in a following PR.

thomastaylor312 commented:
@technosophos I was waiting for the example in https://github.com/kubernetes/charts/pull/1156 to be merged so they matched. If you are ok with not waiting, feel free to merge this

WilliamDenniss commented:
Sorry for the delay. Give me a day and I'll address the comment.

WilliamDenniss commented:
Thanks for your reviews!

@michelleN glad you like my new sample. I addressed your comments.

@thomastaylor312 I'd suggest merging now. The WordPress chart never had any tests in the repo, so updating the docs first won't result in them being out of sync, and the existing doc sample is non-functional, so I think this change is a definitive improvement.

To test kubernetes/charts#1156 yourself:
```
git clone https://github.com/kubernetes/charts.git
cd charts
git fetch origin pull/1156/head:pull1156
git checkout pull1156

helm init
cd charts/stable/wordpress
helm install . --name=wp
helm status wp
# wait for all Deployments to be fully Available
helm test wp
helm delete wp
```

thomastaylor312 commented:
I'll go ahead and merge this
